pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -97,6 +97,7 @@ func (p *staticPolicy) Start(s state.State) error {
 func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Container) (rerr error) {
 	// allocate the memory only for guaranteed pods
 	if v1qos.GetPodQOS(pod) != v1.PodQOSGuaranteed {
+		klog.V(5).InfoS("Memory Allocation Skipped for pod as it is allocated only for Guaranteed pods", "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```

Comment:
similar comments to https://github.com/kubernetes/kubernetes/pull/129685/commits/9d46064fa7f2321b570ae8d2bc8e28be264af0db#r1920259412